src/Microsoft.DotNet.Compiler.Common/ProjectContextExtensions.cs

```diff
@@ -34,25 +37,40 @@ public static void MakeCompilationOutputRunnable(this ProjectContext context, st
 
             if (context.TargetFramework.IsDesktop())
             {
+                // On full framework, copy all dependencies to the output path
                 exporter
                     .GetDependencies()
                     .SelectMany(e => e.RuntimeAssets())
                     .CopyTo(outputPath);
+                    
+                // Generate binding redirects
+                var outputName = context.GetOutputPathCalculator(outputPath).GetAssemblyPath(configuration);
```

Comment:
@davidfowl I know, we chatted a bunch about it. That's not what I am referring to. Now, we pass config/tfm as the output param to makerunnable, and then we pass this output to the calculator and call getassemblypath. We will end up with config/tfm/rid/config/tfm which is not really the right path. In the case of binding redirect, will cause it to fail to find the .config file.

Makes sense?